packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -137,7 +138,12 @@ export class ProfilesCache {
             return validationResult;
         }
 
-        validationResult.port = Number(url.port);
+        if (newUrl.includes(":443")) {
```

Comment:
Maybe we can reuse our new `DEFAULT_PORT` constant here?

I mean, it is super weird string here right now, I'm not sure, that the meaning of it will be remembered...

Reply:
Can we add a unit test(s) for such a behaviour?